src/esp/nav/PathFinder.cpp

```diff
@@ -1349,18 +1350,26 @@ bool PathFinder::Impl::findPathSetup(MultiGoalShortestPath& path,
   if (!path.pimpl_->endRefs.empty())
     return true;
 
+  int numValidPoints = 0;
   for (const auto& rqEnd : path.getRequestedEnds()) {
     dtPolyRef endRef = 0;
     vec3f pathEnd;
     std::tie(status, endRef, pathEnd) =
         projectToPoly(rqEnd, navQuery_.get(), filter_.get());
 
     if (status != DT_SUCCESS || endRef == 0) {
-      return false;
+      ESP_DEBUG() << "Can't project end-point to navmesh, skipping: " << rqEnd;
```

Comment:
If it's really skipping, shouldn't it have a continue logic in the loop?

Reply:
Because the original end-points will be sorted in the next step, we still need an entry for invalid points so the indices will match. This warning message is primarily intended to let the user know that they are providing invalid points in case that is an unexpected bug in their algorithm. The skip here is implicit.

Reply:
Added a way to track this and skip more explicitly.